tests/test_interpolation.py

```diff
@@ -240,20 +240,20 @@ def test_log_eval_N(self):
             assert_almost_equal(act_c, res_c)
         # Full -> \tilde p_0(N) = exp(-N)(exp(N)-1-N)/N^2
         # MMa: Integrate[x^(n-1) (-Log[x]),{x,1/E,1}]
-        p0Nref_full = lambda N, lnx: ((np.exp(N) - 1 - N) / N**2) * np.exp(
+        p0Nref_full = lambda N, lnx: ((np.exp(N) - 1 - N) / N ** 2) * np.exp(
```

Comment:
Here you are using an outdated version of `black`, since 22.1.0 will collapse spaces around `**`, update it in your VS code.

Then I also wonder they have not been fixed by `pre-commit`, might be possible that you forgot to install it?

Reply:
I must have messed up something while merging with develop, thanks for fixing